static-get

```diff
@@ -369,7 +370,26 @@ _static_get() {
                 if [ "${_sget__hash}" = "${_sget__hash_new}" ]; then
                     _cat "${tmpcache}.${uniq_id_mirror}/${_sget__bfile}"
 
-                    if [ "${extract}" ]; then
+                    if [ "${install}" ]; then
+                        _mkdir_p "${directory}/${_sget__bfile%%.tar*}"
```

Comment:
This code seems similar to the one in extract, do you think is it worth to create a function in order to reuse code?